File: src/main/java/com/hohltier/decompiler/menus/MenuBar.java
package com.hohltier.decompiler.menus;

import com.formdev.flatlaf.extras.components.FlatMenuBar;
import com.hohltier.decompiler.menus.help.Help;
import com.hohltier.decompiler.menus.file.File;

public class MenuBar extends FlatMenuBar {

    public MenuBar() {
        add(new File());
        add(new Help());
    }

}

File: src/main/kotlin/moe/sota/decompiler/menus/MenuBar.kt
package moe.sota.decompiler.menus

import com.formdev.flatlaf.extras.components.FlatMenuBar
import moe.sota.decompiler.menus.file.File
import moe.sota.decompiler.menus.help.Help

class MenuBar(
    file: File,
    help: Help
) : FlatMenuBar() {

    init {
        add(file)
        add(help)
    }

}
